assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -148,6 +149,7 @@ protected void configure() {
 
     install(new DevfileModule());
 
+    bind(WorkspaceEntityProvider.class);
```

Comment:
I would say we should have something like WorkspaceModule like DevfileModule. Up2you to decide.